doc/api/buffer.md

```diff
@@ -859,7 +859,7 @@ const buf2 = Buffer.from(buf1);
 buf1[0] = 0x61;
 
 console.log(buf1.toString());
-// Prints: auffer
+// Prints: buffer
```

Comment:
The original was correct. `buf1[0] = 0x61;` replaces the first character in `buf1` with a lower case `a`.